tests/phpunit/tests/media.php

```diff
@@ -4119,6 +4217,8 @@ public function test_the_excerpt_does_not_affect_content_media_count() {
 	 *
 	 * @ticket 56588
 	 *
```

Comment:
```suggestion
	 * @ticket 58235
	 *
```
Added the ticket number that made this change.